documentation/bolt_installing_modules.md

```diff
@@ -273,44 +273,67 @@ requirement.
 > documentation on [Specifying
 > versions](https://puppet.com/docs/puppet/latest/modules_metadata.html#specifying-versions).
 
-## ⛔ Manage modules with a Puppetfile
-
-⛔ **DEPRECATED:** This method of installing and managing modules is deprecated.
-Use the workflows outlined above and avoid editing your Puppetfile.
-
-If you want to install a module to an existing project, use a Puppetfile. This
-method does not automatically resolve module dependencies. If the module you're
-installing requires other modules, make sure you add the required modules to
-your Puppetfile together with the module you're installing.
-
-> **Before you begin**
->
-> - In your Bolt project directory, create a file named `Puppetfile`. 
-> - Add any modules stored locally in `modules/` to the list. For example, 
->   ```puppet
->     mod 'my_awesome_module', local: true
->   ```
->
->   **Bolt deletes any content in `modules/` that is not listed in your
->   Puppetfile.** If you want to keep the content, but you don't want to manage
->   it with the Puppetfile, move the content to a `site-modules` directory in
->   your project.
-
-To install a module:
-   1.  Open Puppetfile in a text editor and add the modules and versions that
-       you want to install. If the modules have dependencies, list those as
-       well. For example:
-       ```puppet
-       # Modules from the Puppet Forge.
-       mod 'puppetlabs-apache', '4.1.0'
-       mod 'puppetlabs-postgresql', '5.12.0'
-       mod 'puppetlabs-puppet_conf', '0.3.0'
-
-       # Modules from a Git repository.
-       mod 'puppetlabs-haproxy', git: 'https://github.com/puppetlabs/puppetlabs-haproxy.git', ref: 'master'
-       ```   
-   1. Run the `bolt puppetfile install` command. Bolt installs modules to the
-      first directory in the `modulepath` setting. By default, this is the
-      `modules/` subdirectory inside the Bolt project directory. To override
-      this location, update the `modulepath` setting in your [project
-      configuration file](bolt_project_reference.md).
+## Manually manage a project's modules
+
+There are a few cases where Bolt is unable to resolve module dependencies and
+manage your project's modules for you. This includes cases when you need to
+do any of the following:
+
+- Install Forge modules from an alternate Forge
+- Install git modules from a repository other than a public GitHub repository
+- Install modules through a proxy
+
+Instead of using Bolt to manage your project's modules, you will need to
+manage your project's Puppetfile manually and use Bolt to install the
+modules listed in the Puppetfile without resolving dependencies.
```

Comment:
I think just a line that "This is the same workflow currently used for the `puppetfile` subcommand, but with the new `module` subcommand" would put users at ease a bit.